package.json

```diff
@@ -1,42 +1,48 @@
 {
-  "name": "nemo-screenshot",
```

Comment:
@grawk I was just trying to make sure that screenshot worked with nemo@2.1.1 and was updating some out of date modules as well 😞